src/components/WebexInMeeting/WebexInMeeting.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import jsonData from '../../data';
+import {WebexJSONAdapter} from '../../adapters';
+
+import {WebexInMeeting, WebexDataProvider} from '..';
+
+// Setup for the stories
+const stories = storiesOf('Webex InMeeting', module);
+const webexAdapter = new WebexJSONAdapter(jsonData);
+const wrapperStyle = {height: '500px', width: '800px', border: '1px solid black'};
+
+stories.add('remote and local media enabled', () => {
+  jsonData.meetings.remoteMedia.remoteAudio = {};
+  jsonData.meetings.remoteMedia.remoteVideo = {};
+  jsonData.meetings.remoteMedia.localAudio = {};
+  jsonData.meetings.remoteMedia.localVideo = {};
+
+  return (
+    <div style={wrapperStyle}>
+      <WebexDataProvider adapter={webexAdapter}>
+        <WebexInMeeting meetingID="remoteMedia" />
+      </WebexDataProvider>
+    </div>
+  );
+});
+
+stories.add('only remote media enabled', () => {
+  jsonData.meetings.remoteMedia.remoteAudio = {};
```

Comment:
You could use `jsonData.meetings['remoteAudio&Video']` here and then you wouldn't need to reset the other one.

Reply:
interstitial doing the same. I guess we need to make the changed in that file as well.